### ISSUE TITLE
Verbosity 

### ISSUE THREAD

**Issue by krober10nd** [CHLNDDEV/OceanMesh2D#187]
* some methods are excessively verbose (msh.clean) for example. 
* perhaps we could handle this in a more consistent way with more controls. For example introducing a verbose kwarg to all methods that is either true or false and false will silence all output while true is left for debugging and advanced use cases.  

* Often warning messages become lost in the sea of text and it’s simply messy.

**Comment by WPringle** [comment]
I think that's a fair comment.

Could we add a m.messages field to show what operations have been performed on it and any notes.

e.g.,

- msh.interp() method was performed on 2030/12/12 12:00 UTC with the following input options: filename = GEBCO, CA = 1, ...
- msh.bound_courant_number() method was performed on 2030/12/12 12:05 UTC with the following input options: dt = 2, cr_max = 1, .... Important note: boundary conditions were deleted during this step.

**Comment by krober10nd** [comment]
I really like the idea of keeping a record in the object indicating in plain English what was done.

**Comment by CHLNDDEV** [comment]
#200
